### PR TITLE
feature: rework driver env header

### PR DESCRIPTION
**Pull request by fireridlle**
### Notes
[ROAD-466 Runtime env tracking](Runtime env tracking)

example
```
'X-Driver-Env': 'driverVersion=4.1.1; driver=nodejs; languageVersion=v12.18.0; env=unknown; os=darwin'
```

### CODE REVIEW

**Review by evbruno — APPROVED** [review]

File: src/_http/index.js [hunk 1]
```diff
@@ -118,25 +118,30 @@ function secretHeader(secret) {
...
+
+  var headers = {
+    'X-FaunaDB-API-Version': packageJson.apiVersion,
   }
 
+  // TODO: api cors must be enabled to accept header X-Driver-Env
+  if (util.isNodeEnv()) {
+    headers['X-Driver-Env'] = Object.keys(driverEnv)
+      .map(key => [key, driverEnv[key].toLowerCase()].join('='))
+      .join('; ')
```
Comment (erickpintor) [inline src/_http/index.js:143]: Are we still supporting ECMA 5? If so, is `.join` compatible?

**Comment by fireridlle** [comment]
update to have a shorter format

File: src/_http/index.js (hunk 1 above)
Comment (fireridlle) [reply src/_http/index.js:145]: this is a standard from the very first ECMA https://www.ecma-international.org/wp-content/uploads/ECMA-262_1st_edition_june_1997.pdf

**Review by erickpintor — APPROVED** [review]
lgtm

**Review by evbruno — APPROVED** [review]

**Pull request merged**